x-pack/plugins/observability_onboarding/public/components/app/custom_logs/wizard/select_logs.tsx

```diff
@@ -0,0 +1,241 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+/* eslint-disable @elastic/eui/href-or-on-click */
+
+import React, { PropsWithChildren, MouseEvent } from 'react';
+import {
+  EuiTitle,
+  EuiLink,
+  EuiButton,
+  EuiFlexGroup,
+  EuiFlexItem,
+  EuiHorizontalRule,
+  EuiSpacer,
+  EuiCard,
+  EuiIcon,
+  EuiIconProps,
+} from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+import {
+  StepPanel,
+  StepPanelContent,
+  StepPanelFooter,
+} from '../../../shared/step_panel';
+import { useWizard } from '.';
+import { useKibanaNavigation } from '../../../../hooks/use_kibana_navigation';
+
+export function SelectLogs() {
+  const { navigateToKibanaUrl, navigateToAppUrl } = useKibanaNavigation();
+  const { goToStep, getState, setState } = useWizard();
+
+  function onBack() {
+    navigateToKibanaUrl('/app/observabilityOnboarding/customLogs');
```

Comment:
the back button here should take us back to the landing page at `/app/observabilityOnboarding`

Reply:
Good catch! Thanks

Reply:
Addressed in [2826d0f](https://github.com/elastic/kibana/pull/156182/commits/2826d0f297aef033a32f9ea6675aea86ce7f5d48).